ABloom/ABloom/Resources/Utilities/ReactionType.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ReactionType.swift
+//  ABloom
+//
+//  Created by 정승균 on 11/8/23.
+//
+
+import Foundation
+
+enum ReactionType: Int {
+  case good = 0
+  case knowEachOther
+  case moreCommunication
+  case moreResearch
+  case error
+  
+  init?(rawValue: Int) {
+    switch rawValue {
+    case 0:
+      self = .good
+    case 1 :
+      self = .knowEachOther
+    case 2 :
+      self = .moreCommunication
+    case 3 :
+      self = .moreResearch
+    default:
+      self = .error
+    }
+  }
```

Comment:
```suggestion
  case good = 0
  case knowEachOther = 1
  case moreCommunication = 2
  case moreResearch = 3
```
이렇게 하는 게 아닌 init으로 하는??이유가 궁금합니다!

Reply:
rawValue로 initialization이 가능하도록 하기 위함입니다!